packages/runner/src/context.ts

```diff
@@ -41,7 +41,8 @@ export function withTimeout<T extends (...args: any[]) => any>(
 
   const { setTimeout, clearTimeout } = getSafeTimers()
 
-  return ((...args: T extends (...args: infer A) => any ? A : never) => {
+  // this function name is used to filter error in test/cli/test/fails.test.ts
+  return (function runWithTimeout(...args: T extends (...args: infer A) => any ? A : never) {
```

Comment:
I'm not sure but I needed to add this filter for `fails` snapshot since `test/cli/fixtures/fails/unhandled.test.ts`'s stacktrace moved a little and now it points to here 

```sh
stderr | unhandled.test.ts > unhandled exception
Error: Uncaught [Error: some error]
    ...
    at file:///home/hiroshi/code/others/vitest/packages/runner/dist/index.js:146:14
    at file:///home/hiroshi/code/others/vitest/packages/runner/dist/index.js:529:11
    at runWithTimeout (file:///home/hiroshi/code/others/vitest/packages/runner/dist/index.js:61:7) Error: some error
    at /home/hiroshi/code/others/vitest/test/cli/fixtures/fails/unhandled.test.ts:8:11
    ...
 ```